experimenter/experimenter/settings.py

```diff
@@ -395,7 +395,7 @@
     },
     "fetch_population_sizing_data": {
         "task": "experimenter.jetstream.tasks.fetch_population_sizing_data",
-        "schedule": 86400,
+        "schedule": crontab(minute=0, hour=6),
```

Comment:
I htought we wanted this to be weekly?

Reply:
That's fine too, I was debating. I'll update to weekly.

Reply:
Ok done. I changed to only execute on Mondays now (the job runs Sunday).